python/private/python_register_toolchains.bzl

```diff
@@ -129,6 +129,12 @@ def python_register_toolchains(
                 )],
             )
 
+        flag_values = tool_versions[python_version].get("flag_values", None)
```

Comment:
Flag values should be passed to `toolchain_aliases` or `toolchain_repos` or both - so that the `target_settings` config settings use them to correctly wire things up.